components/InfoTooltipIcon.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { InfoIcon } from 'lucide-react';
+
+import { cn } from '../lib/utils';
+
+import { Tooltip, TooltipContent, TooltipTrigger } from './ui/Tooltip';
+
+export function InfoTooltipIcon({
+  children,
+  className,
+  size = 16,
+}: {
+  children: React.ReactNode;
+  className?: string;
+  size?: number;
+}) {
+  return (
+    <Tooltip>
```

Comment:
Remove `delayDuration` in this condition, the tooltip is already expected and there's no cost rendering it.

Reply:
Good point, I reduced it to 100ms (default is 500ms) to provide some buffer to prevent opening the tooltip when you just happened to move the mouse over it.